tests/resources/charts/application-connectivity-validator-test/values.yaml

```diff
@@ -0,0 +1,15 @@
+# Default values for application-connectivity-validator-test.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  containerRegistry:
+    path: "europe-docker.pkg.dev/kyma-project"
```

Comment:
Please update repo name to local k3d repository as we do for compass-runtime-agent tests

Reply:
as we agreed offline, this issue will be addressed in separate PR